src/ui/src/builder/BuilderFieldsTools.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+	<div
+		class="BuilderFieldsTools colorTransformer"
+		:data-automation-key="props.fieldKey"
+	>
+		<div class="tools">
+			<div
+				v-for="(tool, toolName) in tools"
+				:key="toolName"
+				class="tool"
+				@click="editTool(toolName)"
+			>
+				<div class="toolName">{{ toolName }}</div>
+				<button class="delete" @click.stop="deleteTool(toolName)">
+					<i class="material-symbols-outlined">delete</i>
+				</button>
+			</div>
```

Comment:
If it's clickable, it should be a button (with `type="button"`, otherwise it sends parent form)
```suggestion
			<button
				v-for="(tool, toolName) in tools"
				:key="toolName"
				class="tool"
				type="button"
				@click="editTool(toolName)"
			>
				<div class="toolName">{{ toolName }}</div>
				<button class="delete" @click.stop="deleteTool(toolName)">
					<i class="material-symbols-outlined">delete</i>
				</button>
			</button>
```

Otherwise, you need to declare more attributes
```suggestion
			<div
				v-for="(tool, toolName) in tools"
				:key="toolName"
				class="tool"
				tabindex="0"
				@click="editTool(toolName)"
				@keydown.enter="editTool(toolName)"
			>
				<div class="toolName">{{ toolName }}</div>
				<button class="delete" @click.stop="deleteTool(toolName)">
					<i class="material-symbols-outlined">delete</i>
				</button>
			</div>
```

Reply:
> otherwise it sends parent form
I don't understand this one

Reply:
Agreed that it needs `tabindex` and `keydown.enter` though

Reply:
>  otherwise it sends parent form

I was referring to the `type` attribute of the `<button>`. If you don't specify the `type="button"`, it's by default `<button type="submit">` ([doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#type)). This can make the button submit the parent form if there is any

```html
<form onsubmit="alert('submited')">
  <input type="submit" value="This button triggers submit form" />
  <button type="submit">This button triggers submit form</button>
  <button>This button triggers submit form</button>

  <button type="button">This button doesn't triggers submit form</button>
</form>
```

Check [this codepen](https://codepen.io/madeindjs/pen/zYgWoWo)

So it's always good to specify `type="button"` to avoid weird behavior

Reply:
Ah I didn't know! Thought it was `button` by default